src/librustc_passes/hir_id_validator.rs

```diff
@@ -1,5 +1,5 @@
-use crate::hir::map::Map;
-use crate::ty::TyCtxt;
+use rustc::hir::map::Map;
+use rustc::ty::TyCtxt;
```

Comment:
Maybe put this file in a `hir` folder if you plan to move the collector there too

Reply:
Hmm; how about doing it if/when the collector is moved? I'm not sure whether I will move it atm.